src/sql/base/browser/ui/table/formatters.ts

```diff
@@ -102,6 +112,14 @@ export function textFormatter(row: number | undefined, cell: any | undefined, va
 	return `<span title="${titleValue}" class="${cellClasses}">${valueToDisplay}</span>`;
 }
 
+
+export function iconCssFormatter(row: number | undefined, cell: any | undefined, value: any, columnDef: any | undefined, dataContext: any | undefined): string {
+	if (isCssIconCellValue(value)) {
+		return `<div role='image' aria-label="${value.ariaLabel}" class="grid-cell-value-container icon codicon slick-icon-cell-content ${value.iconCssClass}"></div>`;
```

Comment:
please make sure the ariaLabel value is escaped

Reply:
Done.